util.sh

```diff
@@ -6,6 +6,7 @@ source ${PEG_ROOT}/colors.sh
 source ${PEG_ROOT}/aws-queries.sh
 
 REM_USER=${REM_USER:=ubuntu}
+DEFAULT_TEMPLATE_FILE=defaults.yml
```

Comment:
Where does defaults.yml get read in? I believe that the instance template file will override these defaults as well.